frontend/public/components/factory/ListPage/ListPageCreate.tsx

```diff
@@ -104,9 +105,14 @@ const ListPageCreate: React.FC<ListPageCreateProps> = ({
     }
   }
 
+  const accessReview: CreateAccessReviewProps = createAccessReview || {
```

Comment:
You could do this without exporting a new type.

```suggestion
  const accessReview: CreateWithPermissionsProps['createAccessReview'] = createAccessReview || {
```